catalog/spatial/kml/spatial-kml-transformer/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,14 +17,14 @@
 
 	<OCD name="Spatial KML Style Map Entry" id="org.codice.ddf.spatial.kml.style">
 
-		<AD name="Attribute Name" id="attributeName" required="true" type="String"
-            description="The name of the Metacard Attribute to match against. e.g. title, metadata-content-type, etc"/>
-
-		<AD name="Attribute Value" id="attributeValue" required="true" type="String"
-            description="The value of the Metacard Attribute."/>
-		
-		<AD name="Style URL" id="styleUrl" required="true" type="String"
-            description="The full qualified URL to the KML Style. e.g. http://example.com/styles#myStyle"/>
+		<AD name="Attribute Name" id="attributeName" type="String"
```

Comment:
❓ `True` is the default value for these attribute definitions so this change doesn't do anything but I still like to have it to make it explicit